### PR TITLE
Added my name in the card, a brief description and link to my Github Profile

### PR DESCRIPTION
**Pull request by Kstheking**
This Pull Request updates the cops_namecards with my name, my brief Introduction and my Github profile link

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -73,10 +73,10 @@ <h3 class="main-title style-count" id="count"></h3>
         <div class="card card__animate">
           <img class="card-img-top img-fluid" data-src="./images/cops.png" alt="Card image cap">
           <div class="card-body">
-            <h5 class="card-title">Title of Name Card</h5>
+            <h5 class="card-title">Khushal Sahni</h5>
```
Comment (jmig5776) [inline index.html]: You are required to create a copy of the namecard. Not to edit the existing.

Comment (Kstheking) [reply index.html]: ok I updated the pull request as per commit 79c86cd

Comment (Kstheking) [reply index.html]: @jmig5776

File: index.html [hunk 2]
```diff
@@ -90,10 +89,22 @@ <h5 class="card-title">Gaurav Burjwal</h5>
...
             <p class="card-text">I'm a Computer Science Undergraduate. Passionate about problem-solving through programming.Machine Learning Enthusiast.</p>
             <a href="https://github.com/gauravburjwal" class="button">GitHub Profile</a>
+
+          </div>
+        </div>
+      </div>
+      
+      <div class="col-12 col-md-6 col-lg-3 contributor">
+        <div class="card card__animate">
+          <img class="card-img-top img-fluid" data-src="./images/cops.png" alt="Card image cap">
```
Comment (jmig5776) [inline index.html]: You didn't attached your image here.

Comment (Kstheking) [reply index.html]: Oh ok, my bad, fixed it as per 3030655 @jmig5776

**Comment by jmig5776** [comment]
Looks good to me.

**Comment by jmig5776** [comment]
Cheers!! Its merged. You successfully completed your challenge.

**Pull request merged**